app/assets/stylesheets/application.scss

```diff
@@ -13,9 +13,14 @@
 @import "pages/index";
 
 body {
-font-family: Circular, -apple-system, BlinkMacSystemFont, Roboto, Helvetica Neue, sans-serif;
+font-family: sans-serif;
```

Comment:
sans-serif is not a font in itself but a font-class ... you could import a font from Google fonts that you can then use